src/compressed_chunk.h

```diff
@@ -0,0 +1,40 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#ifndef COMPRESSED_CHUNK_H
+#define COMPRESSED_CHUNK_H
+
+#include <sys/types.h>      // u_int_t
```

Comment:
small comment, no need for the verbose comments to why you are importing a specific header, those things change a lot and pretty standard.